spec/index/multi_index_spec.rb

```diff
@@ -70,10 +70,17 @@
       end
 
       it "raises SizeError for wrong number of name" do
-        expect { @multi_mi.name = ['n1', 'n2'] }.to raise_error SizeError
-        expect { @multi_mi.name = [ ] }.to raise_error SizeError
-        expect { @multi_mi.name = [''] }.to raise_error SizeError
-        expect { @multi_mi.name = ['n1', 'n2', 'n3', 'n4'] }.to raise_error SizeError
+        error_msg = '\'names\' and \'levels\' should be of same size. Size of the \'name\' array is 2 and size of the MultiIndex \'levels\' and \'labels\' is 3. If you don\'t want to set name for particular level (say level \'i\') then put empty string on index \'i\' of the \'name\' Array.'
```

Comment:
I tried different way of writing `raise_error ` with error msg but locally it is failing. I am searching for the issue in my syntax.